views/databaseIssue.ejs

```diff
@@ -242,9 +254,38 @@
           messageElem.innerText = "Failed to save connection setting. Reason '" + b.reason + "'";
           messageElem.classList = ["text-danger"];
         }
-      });
+      })
+        .finally(() => {
+            getPageElements().restartOctoFarmButton.disabled = notYetValidatedConnection;
+        });
+  }
+  function postRestartServer() {
+      const fetchOptions = getValidatedMongoStringAsFetchOptions();
+      if (!fetchOptions) {
+          return;
+      }
+      fetch("/settings/server/restart", fetchOptions)
+          .then(r => r.json())
+          .catch(e => {
+              const messageElem = getPageElements().connectionResponse;
+              messageElem.innerText = "Could not restart because of lost connection... is OctoFarm still alive and kicking?";
+              messageElem.classList = ["text-danger"];
+          })
+          .then(b => {
+              if (b === true) {
+                  const messageElem = getPageElements().connectionResponse;
+                  messageElem.innerText = "System restart command was successful,the server will restart in 5 seconds... Please refresh page or wait for automatic refresh in 10 seconds.";
+                  messageElem.classList = ["text-success"];
+                  setTimeout( () => location.reload(), 10 * 1000 )
+              }
+              else {
+                  const messageElem = getPageElements().connectionResponse;
+                  messageElem.innerText = "Failed to Restart OctoFarm. Reason:'" + b + "'";
+                  messageElem.classList = ["text-danger"];
+              }
+          });
```

Comment:
Pretty much followed your format here for the page. 

I don't return a message from the original function so just added it in as a string. 

I re-used your fetchOptions too as I dis-regard what's send in the body to the endpoint